bskyweb/cmd/bskyweb/server.go

```diff
@@ -60,21 +71,37 @@ func serve(cctx *cli.Context) error {
 	xrpcc.Auth.Did = auth.Did
 	xrpcc.Auth.Handle = auth.Handle
 
-	server := Server{xrpcc}
+	// httpd
+	var (
+		httpTimeout          = 10 * time.Minute
+		httpMaxHeaderBytes   = 10 * (1024 * 1024)
```

Comment:
These seem a bit long and a bit large to me. I would do 2 minutes and 2 MByte? The header size is just bikeshedding, but I feel like a 10 minute timeout could lead to some very strange and hard to debug/repro situations.